### PR TITLE
[DO-2123]Fix Diagram legend language

### PR DESCRIPTION
**Pull request by AlenaYanish**

### CODE REVIEW

File: src/components/pages/dashboard/KvedChart.jsx [hunk 1]
```diff
@@ -58,7 +67,7 @@ const KvedChart = () => {
       <canvas className="mt-3" id="top-kved-chart" height="115" />
       <PieChartLegend
         items={topKvedData.map((el) => ({
-          label: `${el.kved.code}  ${upFirstLetter(el.kved.name)}`,
+          label: `${el.kved.code}  ${getName(el.kved)}`,
```
Comment (roman-tiukh) [inline src/components/pages/dashboard/KvedChart.jsx:70]: use `getLocaleField`

File: src/components/pages/datasets/kved/KvedList.jsx [hunk 2]
```diff
@@ -17,7 +16,7 @@ const KvedList = () => {
     {
       header: t('class'),
       defaultSelected: true,
-      prop: 'name',
+      prop: i18n.language === 'uk' ? 'name' : 'name_en',
```
Comment (roman-tiukh) [inline src/components/pages/datasets/kved/KvedList.jsx:19]: use `getLocaleField`

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/pages/datasets/kved/KvedList.jsx (hunk 2 above)
Comment (Andrii17s) [reply src/components/pages/datasets/kved/KvedList.jsx:19]: It's done here: https://github.com/3v-workspace/DataOcean/pull/195/files

File: src/components/pages/datasets/kved/KvedList.jsx [hunk 3]
```diff
@@ -5,9 +5,8 @@ import PageBox from 'components/pages/PageBox';
 import { useTranslation } from 'react-i18next';
 import { getLocaleField } from 'utils';
 
-
 const KvedList = () => {
-  const { t } = useTranslation();
+  const { t, i18n } = useTranslation();
```
Comment (roman-tiukh) [inline src/components/pages/datasets/kved/KvedList.jsx:9]: unused var?

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**